frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -133,11 +133,18 @@ export function MissionQueueView({ refreshInterval }: RefreshProps) {
     }, [missionQueue, ongoingMissions])
 
     const missionQueueDisplay = missionQueue.map(function (mission, index) {
```

Comment:
```suggestion
    const missionQueueDisplay = missionQueue.map((mission, index) =>
```

Reply:
Just thought this was a good opportunity to make the style consistent with the rest of the code